src/main/kotlin/day16/repository/ImageRepositoryImpl.kt

```
@@ -0,0 +1,58 @@
+
+
+            val elapsedTime = endTime.nano - startTime.nano
+            val duration = Duration.between(startTime, endTime)
+
+            val durationTIme = LocalTime.ofNanoOfDay(duration.toNanos())
+            val foramttedDuration = durationTIme.format(DateTimeFormatter.ofPattern("HH:mm:ss.SSSSSSS"))
+
+
+
+            println("소요시간 : $foramttedDuration")
+            println("용량 : ${String.format("%.2f", sizeInKb)} KB")
```

detekt (1.23.7)

[warning] String.format("%.2f", sizeInKb) uses implicitly default locale for string formatting.

(detekt.potential-bugs.ImplicitDefaultLocale)